### ISSUE TITLE
Parse Setup

### ISSUE THREAD

**Issue by nanoxd** [adampash/nag_bot#1]
Hi,

Thanks for building this. I was looking to build something like this and I came across your project. I tried setting up the project using the included readme but encountered a few issues. In the config file, I didn't understand what `token` I should input. I realized that it was the slash command token after fiddling with it. 

I was able to get it set up but I don't get any things delivered when the job is scheduled. I get the message logged that it will send a message but when it comes time to run the jobs, it does nothing. I was wondering if you could expand on getting everything setup with Parse and Slack Slash commands

**Comment by adampash** [comment]
Hi @nanoxd — sorry for the very slow response! Were you ever able to set it up?

**Comment by nanoxd** [comment]
No, I ended up getting busy with other projects and never got around to fixing it. Any tips on getting it set up?
